tests/ha/single_pod_restart/test_data_pod_restart_api.py

```diff
@@ -1012,3 +1013,237 @@ def test_copy_object_during_pod_restart(self):
+                                                       "deployment_backup":
+                                                           self.deployment_backup,
+                                                       "num_replica": self.num_replica,
+                                                       "set_name": self.set_name},
+                                       clstr_status=True)
+        LOGGER.debug("Response: %s", resp)
+        assert_utils.assert_true(resp[0], f"Failed to restore pod by {self.restore_method} way or "
+                                          f"cluster status is not good")
+        LOGGER.info("Step 5: Successfully restarted the pod with replica method and checked "
+                    "cluster status")
+        self.restore_pod = False
+        event.clear()
+        thread.join()
+
+        LOGGER.info("Step 6: Verify responses from background process")
+        responses = tuple()
+        while len(responses) < 3:
+            responses = output.get(timeout=HA_CFG["common_params"]["60sec_delay"])
+        if not responses:
+            assert_utils.assert_true(False, "Failure in background process")
+        checksums = responses[1]
+        exp_fail_count = responses[2]
+        assert_utils.assert_true(responses[0],
+                                 "Failures observed in overwrite or download background process")
+        assert_utils.assert_false(exp_fail_count, "Failures observed in overwrite or download "
+                                                  "in background process while event was set")
+        for checksum in checksums.values():
+            assert_utils.assert_equal(checksum[0], checksum[1],
+                                      f"Checksum does not match, Expected: {checksum[0]} "
+                                      f"Received: {checksum[1]}")
+        LOGGER.info("Step 6: Successfully verified responses from background process")
+
+        LOGGER.info("Step 7: Read-Verify already overwritten object in healthy cluster")
+        download_path = os.path.join(self.test_dir_path, f"{self.object_name}_download.txt")
+        resp = self.s3_test_obj.object_download(self.bucket_name, self.object_name, download_path)
+        assert_utils.assert_true(resp[0], f"Object download failed. Response: {resp[1]}")
+        dnld_checksum = self.ha_obj.cal_compare_checksum([download_path], compare=False)[0]
+        system_utils.remove_file(file_path=download_path)
+        assert_utils.assert_equal(upld_chcksm, dnld_checksum, "Upload & download checksums don't "
+                                                              f"match. Expected: {upld_chcksm} "
+                                                              f"Actual: {dnld_checksum}")
+        LOGGER.info("Step 7: Successfully Read-Verify already overwritten object in healthy "
+                    "cluster")
+
+        t_t = int(perf_counter_ns())
+        bucket_name = f"bucket-{t_t}"
+        object_name = f"object-{t_t}"
```

Comment:
DTM flag.

Reply:
instead of adding DTM flag I guess lets create all required buckets beforehand and then use them

Reply:
> instead of adding DTM flag I guess lets create all required buckets beforehand and then use them

Updated as per test step @sonalk0209 @apurwamohite 
Step 8 is about overwriting existing bucket objects.

Reply:
Updated.